File: DSA/RATNA_KHANNA_590013141/DAY_32_QUESTION_2.c
#include <stdio.h>

int main() {
    int n, i, result = 0;
    printf("Enter number of elements: ");
    scanf("%d", &n);
    int arr[n];
    printf("Enter the elements:\n");
    for(i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
        result ^= arr[i];  
    }
    printf("The non-repeating element is: %d\n", result);
    return 0;
}

File: DSA/VenikaPanwar_590013755/Day_32_Question_2.java
import java.util.*;

public class FDay_32_Question_2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number of elements: ");
        int n = sc.nextInt();

        int[] arr = new int[n];
        System.out.println("Enter elements (where one is non-repeating):");
        for (int i = 0; i < n; i++) arr[i] = sc.nextInt();

        int result = 0;
        for (int val : arr) result ^= val;

        System.out.println("Unique non-repeating element: " + result);
    }
}
